### PR TITLE
Fix web_base.py

### PR DESCRIPTION
**Pull request by zomchak-code**
Fix for bug in SitemapLoader

`aiohttp` `get` does not accept `verify` argument, and currently throws error, so SitemapLoader is not working

This PR fixes it by removing `verify` param for `get` function call

Fixes #6107

#### Who can review?

Tag maintainers/contributors who might be interested:

@eyurtsev

### CODE REVIEW

File: langchain/document_loaders/web_base.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ async def _fetch(
             for i in range(retries):
                 try:
                     async with session.get(
-                        url, headers=self.session.headers, verify=self.verify
```
Comment (dev2049) [inline langchain/document_loaders/web_base.py:113]: seems like there is a verify_ssl kwarg, could that be what this is supposed to be?

Comment (pandemosth) [reply langchain/document_loaders/web_base.py:113]: Yes.

From aiohttp docs:
verify_ssl – Perform SSL certificate validation for HTTPS requests (enabled by default). May be disabled to skip validation for sites with invalid certificates.

This bug is also impacting WebBaseLoader when calling async, fyi.

**Review by arunchinnachamy — CHANGES_REQUESTED** [review]
the parameter needs to be removed from Line No. 188 as well in the function scrape.

https://github.com/hwchase17/langchain/pull/6519/files/bf66b441be6496cf1d4227bd54d2b941433f5580#diff-8d822032870450954e168387e16257457d811a2539a5e94fa9317865f88dde71L188

File: langchain/document_loaders/web_base.py (hunk 1 above)
Comment (techcenary) [reply langchain/document_loaders/web_base.py:113]: @dev2049 @pandemosth 

`verify_ssl` arg has been deprecated, and replaced with:

ssl –
SSL validation mode. None for default SSL check
([ssl.create_default_context()](https://docs.python.org/3/library/ssl.html#ssl.create_default_context) is used), False for skip SSL certificate validation, [aiohttp.Fingerprint](https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.Fingerprint) for fingerprint validation, [ssl.SSLContext](https://docs.python.org/3/library/ssl.html#ssl.SSLContext) for custom SSL certificate validation.
Supersedes verify_ssl, ssl_context and fingerprint parameters.
(From aiohttp docs: https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientSession.request)

I've updated the code to use `ssl` param

**Comment by techcenary** [comment]
@arunchinnachamy 
> the parameter needs to be removed from Line No. 188 as well in the function scrape.

Line 188 is using `self.session` which is a `requests` [session object](https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification)
I've refactored the code and moved this param setting in the constructor

**Review by zomchak-code — COMMENTED** [review]
@dev2049 @pandemosth @arunchinnachamy Comments addressed

**Comment by zomchak-code** [comment]
@rlancemartin, @eyurtsev
Let me know what I can do to help merge this

**Comment by zomchak-code** [comment]
@rlancemartin, @eyurtsev
I fixed the linting error

**Comment by zomchak-code** [comment]
@rlancemartin, @eyurtsev
Sorry for spamming, but this is a small fix for important bug. I currently can not use langchain in my workflow because of it.
Just let me know what can I do to get this merged faster

**Comment by rlancemartin** [comment]
> @rlancemartin, @eyurtsev Sorry for spamming, but this is a small fix for important bug. I currently can not use langchain in my workflow because of it. Just let me know what can I do to get this merged faster

Thanks! Running tests now and will merge once it passes.

**Pull request merged**